src/docfx/build/metadata/SystemMetadata.cs

```diff
@@ -59,6 +59,9 @@ internal class SystemMetadata
     [JsonProperty("_op_canonicalUrlPrefix")]
     public string? CanonicalUrlPrefix { get; set; }
 
+    [JsonProperty("_op_canonicalUrl")]
+    public string? OpCanonicalUrl { get; set; }
```

Comment:
what does this metadata used for?

Reply:
`_op_canonicalUrl is in the blob content, which is later used in HTML tags. `

please check description of work item 664227

Reply:
Before, the _op_canonicalUrl is generated by JINT template js directly but not defined on DocFX